lib/models/addon.js

```diff
@@ -307,6 +335,18 @@ var Addon = CoreObject.extend({
    */
   _warn: warn,
 
+  _emberCLIBabelConfigKey: function() {
+    // future versions of ember-cli-babel will be moving the location for its
+    // own configuration options out of `babel` and will be issuing a deprecation
+    // if used in the older way
+    //
+    // see: https://github.com/babel/ember-cli-babel/pull/105
+    var emberCLIBabelInstance = findAddonByName(this.addons, 'ember-cli-babel');
```

Comment:
Agreed, but I wanted to decouple new public API's from this PR. 

Technically, `project.findAddonByName` is also private (per YUIDocs IIRC), I'd suggest a future RFC to make that method public on both `Project` and `Addon`.